mobile/src/modules/auth/screens/HomeScreen.tsx

```diff
@@ -1,70 +1,69 @@
 import React from 'react';
-import { StyleSheet, View, Picker, Button } from 'react-native';
+import { core } from '../translations';
+
+import { Button, View } from 'react-native';
 import { NavigationInjectedProps, withNavigation } from 'react-navigation';
 import { connect } from 'react-redux';
-import { Text, TextThemes } from '../../../core/components/base/Text';
+import { AuthReduxState } from '../store/reducer';
+
+import { Text } from '../../../core/components/base/Text';
 import { HeaderTitle } from '../../../core/components/header/HeaderTitle';
 import { i18n } from '../../../core/i18n';
 import { InjectedReduxProps } from '../../../core/interfaces';
 import { FullHeightView } from '../../../core/layout/FullHeightView';
 import { Screen } from '../../../core/layout/Screen';
-import { ApplicationState } from '../../../core/store/initialState';
-import { Sizes } from '../../../core/theme/sizes';
-import { AuthReduxState } from '../store/reducer';
-import { core } from '../translations';
 import { routes } from '../../../core/navigation/routes';
+import { ApplicationState } from '../../../core/store/initialState';
 
 interface IProps extends InjectedReduxProps {
-    auth: AuthReduxState,
+  auth: AuthReduxState,
 }
 type Props = IProps & NavigationInjectedProps;
 
-const styles = StyleSheet.create({
-    hintView: {
-        paddingTop: Sizes.SMALL,
-        flexGrow: 1
-    }
-});
-
 class Home extends React.PureComponent<Props> {
-    static navigationOptions = {
-        headerTitle: <HeaderTitle>{i18n.t(core.screens.home.title)}</HeaderTitle>
-    };
 
-    changeAccount() {
-        this.props.navigation.navigate(routes.accounts);
-    }
+  constructor (props: Props) {
+    super(props);
+    this.changeAccount = this.changeAccount.bind(this);
```

Comment:
just use arrow functions below for any non-lifecycle methods like `handleChangeAccount = () => { code; }`